pkg/events/derive/hooked_syscall.go

```diff
@@ -53,7 +55,14 @@ func deriveDetectHookedSyscallArgs(kernelSymbols *environment.KernelSymbolTable)
 		syscallName := convertToSyscallName(syscallId)
 		hexAddress := fmt.Sprintf("%x", address)
 
-		hookedFuncSymbols, err := kernelSymbols.GetSymbolByAddr(address)
+		var hookedFuncSymbols []*environment.KernelSymbol
+		err = capabilities.GetInstance().Specific(
+			func() error {
+				var capErr error
+				hookedFuncSymbols, capErr = kernelSymbols.GetSymbolByAddr(address)
+				return capErr
+			},
+			cap.SYSLOG) // Required to read /proc/kallsyms
```

Comment:
as we don't return the error back, only a fixed value, and error as nil, perhaps we should log.Error if error is differen than nil?

Reply:
What do you mean we don't return the error back? isn't capErr what we get in err back?

Reply:
Ok, now I see what you mean. This part of the code wasn't change in this PR and we didn't return the error from GetSymbolByAddr before as well. Anyway, this is a good point. I'll add this